### PR TITLE
[Feature] Support Wide ResNet

### PR DESCRIPTION
**Pull request by yasu0001**
## Motivation
Based on #678 

## Modification
Add two wide resnet config files for ImageNet.

### CODE REVIEW

**Comment by yasu0001** [comment]
I've confirmed that the accuracy is almost same after model conversion.
The accuracy of torchvision is cited from https://pytorch.org/vision/stable/models.html

|Model|Accs@1|Accs@1(torchvision)|Accs@5|Accs@5(torchvision)|
|:--:|:--:|:--:|:--:|:--:|
|Wide ResNet 50|77.7| 78.47|93.79| 94.09|
|Wide ResNet 101|78.18|78.85|93.98|94.28|
|ResNet 50|75.36|76.130|92.52|92.862|

I would like to ask you about CLA.
The status of CLA is still pending after agreement the License.
This may be because I used wrong mail address in `git.config` when I pushed.
What I should do in this case?

**Comment by Ezra-Yu** [comment]
> I've confirmed that the accuracy is almost same after model conversion. The accuracy of torchvision is cited from https://pytorch.org/vision/stable/models.html

Actually, the accuracy should be the same. So there is something different with setting in torchvision. Maybe it causes by Resize backend, You can try:
```
dict(type='Resize', size=(256, -1), backend='pillow'),
```



> This may be because I used wrong mail address in git.config when I pushed.

Did you mean that you have pushed your commits before config your email address, which caused the username unable to be tracked?  Maybe you need to use `git rebase` to modify your commit information. Or just close this PR and re-create a new one.

**Comment by yasu0001** [comment]
> Actually, the accuracy should be the same. So there is something different with setting in torchvision. Maybe it causes by Resize backend, You can try:
`dict(type='Resize', size=(256, -1), backend='pillow'),`

I confirmed the accuracy is the same as torchvision by modifying resize backend.

|Model|Accs@1|Accs@1(torchvision)|Accs@5|Accs@5(torchvision)|
|:--:|:--:|:--:|:--:|:--:|
|Wide ResNet 50|78.48|78.47|94.08| 94.09|
|Wide ResNet 101|78.84|78.85|94.28|94.28|
|ResNet 50|76.14|76.13|92.87|92.86|

> Did you mean that you have pushed your commits before config your email address, which caused the username unable to be tracked? Maybe you need to use git rebase to modify your commit information. Or just close this PR and re-create a new one.

Yes, I modified commit message.

**Review by mzr1996 — APPROVED** [review]
LGTM

**Comment by Davidgzx** [comment]
No support for 3 stage WRN? like WRN 16 to WRN 40?

**Comment by Ezra-Yu** [comment]
@Davidgzx  Can you attach the code and weights link?

**Comment by Ezra-Yu** [comment]
@Davidgzx In the official repo, the wrn16 or the wrn40 are trained by using cifra10 and cifra100. I think the ImagNet-1k dataset is more general.

**Comment by OpenMMLab-gd** [comment]
Hi @yasu0001！First of all, we want to express our gratitude for your significant PR in the MMClassification project. Your contribution is highly appreciated, and we are grateful for your efforts in helping improve this open-source project during your personal time. We believe that many developers will benefit from your PR.

We would also like to invite you to join our Special Interest Group (SIG) private channel on Discord, where you can share your experiences, ideas, and build connections with like-minded peers. To join the SIG channel, simply message moderator— OpenMMLab on Discord or briefly share your open-source contributions in the #introductions channel and we will assist you. Look forward to seeing you there! Join us ：https://discord.gg/UjgXkPWNqA

If you have WeChat account，welcome to join our community on WeChat. You can add our assistant ：openmmlabwx. Please add "mmsig + Github ID" as a remark when adding friends：）
Thank you again for your contribution❤ @yasu0001
